### DISCUSSION TITLE
待在家就可以發出國玩的限動！當 Instagram 結合 Emu 會產生怎樣的火花？

### DISCUSSION THREAD

**Discussion by BeaGle0928** [1101-GS4719-C/Python-Community-GS4719-C#1632] [科技新知]
Meta 近期於 Meta Connect 中推出一系列 AI 工具和平台，而我對於其中兩個項目相當感興趣，分別是Emu 導入Instagram 以及 Meta AI 名人聊天機器人，而我將分為兩篇文章來分享。
圖像生成模型 Emu (Expressive Media Universe），輸入文字指令即可產出圖像。這聽起來不是一個太新奇的AI功能，然而當我們將這樣的技術引入我們平時使用的社群軟體…。Meta 為 Instagram 推出照片編輯功能「Restyle」和 「Backdrop」。

Restyle 將依照使用者指令所描述的視覺風格來改造圖片。例如輸入「水彩」或「報章雜誌的拼貼圖，邊緣撕紙效果」等描述，就可讓照片呈現想像中的外觀與風格。

Backdrop 則為風景或背景編輯功能，同時運用影像分割基礎模型 Segment Anything Model 的學習結果。輸入「在我背後加上超美極光」或「被一群小狗圍繞在身邊」等文字提示後，AI 工具會將照片中的主角置於前景，背景即為剛才形容的畫面。

我認為將圖片生成模型導入 Instagram可能不是一個太好的想法，儘管它可以讓人們隨心所欲的創作屬於自己的圖片，但這樣的功能也一定程度上影響了Instagram 最一開始「分享生活」的初衷。
想問大家覺得這個功能好用嗎？Instagram真的需要這樣的功能嗎？

資料來源:https://www.inside.com.tw/amparticle/32920-meta-connect-2023-ai?fbclid=IwAR0NehHcuycdIWAQyCKpY3MWQm6r4kRfCrpd-U-HANtKiNAzmUAg_b4Yg3Q

**Comment by yxin214** [comment]
我認同你的觀點！Instagram除了用來通訊交友外，更重要的是他即時分享動態的功能。即便是不常見面的朋友或很久沒聊天的同學，我們能透過Instagram的動態去交流彼此的生活。若背景編輯功能普及，使用Instagram又有什麼意義？一張美麗的風景照是大自然的鬼斧神工，不該是人工智慧的巧奪天工。我還是希望大家使用Instagram分享最真實的生活，那也是大家所想看見的吧？
